scripts/local_environment.sh

```diff
@@ -53,22 +65,22 @@ function destroy_container() {
     fi
 }
 
-function main() {
+function parse_arguments() {
     while [[ "$#" -gt 0 ]]; do
         case "$1" in
             --deploy)
-                check_container_engine
-                start_container
+                RUN_COMMAND="deploy"
                 shift
                 ;;
             --destroy)
-                check_container_engine
-                destroy_container
+                RUN_COMMAND="destroy"
                 shift
                 ;;
-            --get-runtime)
-                check_container_engine
-                shift
+            --runtime)
+                if [[ -n "$2" ]]; then
+                    RUNTIME="$2"
+                    shift 2
+                fi
```

Comment:
This needs to fail if there is no `$2`, otherwise the script will get stuck in a loop ;-).

Reply:
Added failure in case no value provided.